common/persistence/cassandra/cassandraPersistenceTest.go

```diff
@@ -77,7 +76,7 @@ func NewTestCluster(keyspace, username, password, host string, port int, schemaD
 		Hosts:          host,
 		Port:           port,
 		MaxConns:       2,
-		ConnectTimeout: 600 * time.Millisecond,
+		ConnectTimeout: 30 * time.Second,
```

Comment:
no wonder ... 600ms!

Reply:
The 600ms was ignored as it was not correctly pass down to the destination. The default was 10s, so we were using 10s. I was able to repro the failure with 10s on my local a few times. So far, 30s seems to be working for me on my local.

Reply:
btw, where is the change that makes this get passed through appropriately?

Reply:
Line 153 in this PR.